routes/home.js

```diff
@@ -2,11 +2,14 @@ const shuffle = require('knuth-shuffle-seeded')
 const apps = require('../lib/apps')
 const featuredCompanies = require('../lib/featured-companies')
 
+const showAnnounceBanner = new Date() < new Date(2020, 1, 1)
```

Comment:
Could this be stored in the `.yml` alongside the content of the announcement banner?

Reply:
I think it would strange to have an untranslatable string in `data/locale.yml`.

Reply:
Makes sense. I'm fine with that! I find it a bit too "hard-coded" that the date variable is not set in a place that's intuitive for configuration purposes, but I'm not familiar with the website codebase so don't have any suggestions to improve on this.